lib/peep.rb

```diff
@@ -0,0 +1,14 @@
+require 'pg'
+
+class Peep
+  def self.all
+    if ENV['ENVIRONMENT'] = 'test'
```

Comment:
could reference that external database object again here, would help DRY code up a bit as well as encapsulate